docs/visualize/aggregations.asciidoc

```diff
@@ -0,0 +1,101 @@
+[[supported-aggregations]]
+=== Supported aggregations
+
+All of these visualizations support:
+
+**Metric Aggregations:**
+
+* {ref}/search-aggregations-metrics-avg-aggregation.html[Average]
+* {ref}/search-aggregations-metrics-valuecount-aggregation.html[Count]
+* {ref}/search-aggregations-metrics-max-aggregation.html[Maximum]
+* *{ref}/search-aggregations-metrics-percentile-aggregation.html[Median]*: The value that is in the 50% percentile.
+* {ref}/search-aggregations-metrics-min-aggregation.html[Minimum]
+* {ref}/search-aggregations-metrics-sum-aggregation.html[Sum]
+* Unique Count: Uses the {ref}/search-aggregations-metrics-cardinality-aggregation.html[Cardinality] aggregation.
+* Standard Deviation: Uses the {ref}/search-aggregations-metrics-extendedstats-aggregation.html[_extended stats_]
+aggregation.
+* Top Hit: Uses the {ref}/search-aggregations-metrics-top-hits-aggregation.html[_top hits_]
+aggregation to return a sample of individual documents. If more than one document is matched by this aggregation, you must choose a technique for combining the values. These techniques are average, min, max, and sum.
+* Percentiles: The {ref}/search-aggregations-metrics-percentile-aggregation.html[_percentile_] aggregation divides the
+values in a numeric field into percentile bands that you specify. Select a field from the drop-down, then specify one
```

Comment:
Is it necessary to keep the task content in these definitions? I think we can remove `Select a field from the drop-down, then specify one or more ranges in the *Percentiles* fields. Click the *X* to remove a percentile field. Click *+ Add* to add a percentile field.` and relocate it at point of use.